rallyrolebot/cogs/defaults_cog.py

```diff
@@ -0,0 +1,86 @@
+import json
+import sys
+import traceback
+
+import discord
+from discord.ext import commands, tasks
+from discord.utils import get
+
+import data
+import rally_api
+import validation
+import errors
+
+from cogs import update_cog
+
+from constants import *
+from utils import pretty_print
+
+
+class DefaultsCommands(commands.Cog):
```

Comment:
#1 mentions that there should be an __unregister_user__ command for the bounty.

Reply:
Commands for unregistering are in `rally_cog.py`

Reply:
Whoops, my bad!